packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig

```diff
@@ -56,6 +56,11 @@
 {%- set _mainPropsWithoutId = props | filter((value, prop) => prop != 'id') -%}
 {%- set _allowedAttributes = [ 'autocomplete' ] -%}
 
+{# Deprecations #}
+{% if _validationState === 'error' %}
+    {% deprecated _rootClassName + ': This "error" value for "validationState" property will be renamed to "danger" in the next major version. Use this instead.️️' %}
```

Comment:
```suggestion
    {% deprecated _rootClassName + ': The "error" value for "validationState" property will be renamed to "danger" in the next major version.' %}
```

But I am not sure, it just sounds better to me. If it is ok, please change it everywhere (even in the react package).

Reply:
fixed